ykutil/src/addr.rs

```diff
@@ -11,15 +11,20 @@ use std::{
     ptr::null,
 };
 
+pub fn dladdr(vaddr: usize) -> Result<Dl_info, ()> {
```

Comment:
A short doc string here might be useful (e.g. for the `Err` case).

Reply:
Fixed in dfda6853f96232217fd0927ce41039281e711e10